projects/demo/src/modules/components/input/examples/import/import.md

```diff
@@ -1,6 +1,6 @@
 ```ts
 import {ReactiveFormsModule} from '@angular/forms';
-import {TuiInputModule} from '@taiga-ui/kit';
```

Comment:
nit: we need to change the chip in the demos as well 
<img width="188" alt="Снимок экрана 2024-05-24 в 21 35 15" src="https://github.com/taiga-family/taiga-ui/assets/46284632/086c18d6-b13e-4749-bd27-aed9686790fa">

Reply:
I don't want to do it because we will eventually replace all those controls in kit with updated versions, presumably before 4.0. I only updated import instructions so I don't miss any imports :)